spec/controllers/submissions_controller_spec.rb

```diff
@@ -0,0 +1,48 @@
+require 'rails_helper'
+
+RSpec.describe SubmissionsController do
+  include ApiHelpers
+
+  render_views
+
+  let(:email) { 'email@example.com' }
+  let(:user) { User.create!(email: email, uid: 'auth0|123', auth_hash: { uid: 'auth0|123', info: { email: email } }) }
```

Comment:
`FactoryBot.create(:user)` already exists for creating users in tests - might want to amend the factory to generate the `auth_hash` attribute if necessary.